### DISCUSSION TITLE
Offline documentation availability?

### DISCUSSION THREAD

**Discussion by NoahNxT** [nWidart/laravel-modules#1783] [Q&A] [answered]
Hi!

I'm traveling a lot and there is not always a possibility to have any connection to the internet. 
I'm trying to collect some documentation for soon to happen travels.

Is there any way that there is an offline version we could download of the documentation so when I'm off the grid I still can work on my codebase with docs?

Something like they're doing for Laravel for example (I guess it's not official but very handy when abroad)
https://github.com/mnishihan/laravel-docs-in-pdf

Kind regards!

**Comment by dcblogdev** [comment]
hmm that's a good shout, I have all the docs in Markdown so its just a case of putting it all into a PDF, I'll sort something out.

**Reply by dcblogdev** [reply, marked as answer]
Here's a PDF version of the docs for v11 https://github.com/laravel-modules-com/docs/blob/main/v11/laravel-modules-v11.pdf

**Reply by NoahNxT** [reply]
Allright! Big thanks for the effort!

**Comment by kingtracker141414** [comment]
yes there are way to do that

**Reply by dcblogdev** [reply]
yes see above, also linked to the source for the PDF to be generated https://github.com/laravel-modules-com/docs-pdf/tree/main/export
